.github/workflows/main.yml

```diff
@@ -393,20 +393,23 @@ jobs:
       - name: run test suite (Git WD, standard)
         run: |
           bin/ch-test all
+          [[ $(cat /tmp/ch-test.tmp.$USER/rootemu) == "no" ]]
```

Comment:
1. Add a brief comment explaining why we're validating that this is `no`.
2. No quotes needed here.

Reply:
Also while `==` is correct, it’s slightly obscure — equivalent to `=`, and POSIX only supports `=`. Consider changing to `=`.